flycheck-phpstan.el

```diff
@@ -51,14 +51,18 @@
       (when (and phpstan-flycheck-auto-set-executable
                  (not (and (boundp 'flycheck-phpstan-executable)
                            (symbol-value 'flycheck-phpstan-executable)))
-                 (or (eq 'docker phpstan-executable)
-                     (and (consp phpstan-executable)
-                          (stringp (car phpstan-executable))
-                          (listp (cdr phpstan-executable)))))
+                 (or (stringp phpstan-executable)
+                     (eq 'docker phpstan-executable)
+                     (and (eq 'root (car-safe phpstan-executable))
+                          (stringp (cdr-safe phpstan-executable)))
+                     (and (stirngp (car-safe phpstan-executable))
```

Comment:
Typo https://github.com/emacs-php/phpstan.el/issues/38

Reply:
Sorry, I have to check more.